azure-cli2017.pyproj

```diff
@@ -130,809 +130,896 @@
     <Compile Include="azure-cli-testsdk\azure\cli\testsdk\utilities.py" />
     <Compile Include="azure-cli-testsdk\azure\cli\testsdk\__init__.py" />
     <Compile Include="azure-cli-testsdk\setup.py" />
+    <Compile Include="azure-cli\azure\cli\command_modules\acr\build.py" />
```

Comment:
The azure-cli2017.pyproj file did not contain the new command module file directories so I added them. Let me know if this was intended and I'll remove the change.